cmd/kubeadm/app/cmd/upgrade/diff_test.go

```diff
@@ -124,3 +136,73 @@ func TestRunDiff(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateManifests(t *testing.T) {
+	rand.Seed((time.Now().UnixNano()))
```

Comment:
we can remove this

Reply:
Thanks your comments. @neolit123 
It's all done.